Ports/quake3/patches/0001-Meta-Refactor-Makefile-to-support-Serenity.patch

```diff
@@ -0,0 +1,142 @@
+From 10ab889934f27a71af9029f52e8f9558efe73c3b Mon Sep 17 00:00:00 2001
+From: Jesse Buhagiar <jooster669@gmail.com>
+Date: Fri, 25 Mar 2022 09:39:21 +1100
+Subject: [PATCH 1/8] Meta: Refactor Makefile to support Serenity
+
+---
+ Makefile | 39 ++++++++++++++++++++-------------------
+ 1 file changed, 20 insertions(+), 19 deletions(-)
+
+diff --git a/Makefile b/Makefile
+index 9e535559..ea2db896 100644
+--- a/Makefile
++++ b/Makefile
+@@ -3,8 +3,8 @@
+ #
+ # GNU Make required
+ #
+-COMPILE_PLATFORM=$(shell uname | sed -e 's/_.*//' | tr '[:upper:]' '[:lower:]' | sed -e 's/\//_/g')
+-COMPILE_ARCH=$(shell uname -m | sed -e 's/i.86/x86/' | sed -e 's/^arm.*/arm/')
++COMPILE_PLATFORM=serenity
++COMPILE_ARCH=amd64
+ 
+ #arm64 hack!
+ ifeq ($(shell uname -m), arm64)
+@@ -35,13 +35,13 @@ ifndef BUILD_GAME_QVM
+   BUILD_GAME_QVM   =
+ endif
+ ifndef BUILD_BASEGAME
+-  BUILD_BASEGAME =
++  BUILD_BASEGAME =1
+ endif
+ ifndef BUILD_MISSIONPACK
+-  BUILD_MISSIONPACK=
++  BUILD_MISSIONPACK=0
+ endif
+ ifndef BUILD_RENDERER_OPENGL2
+-  BUILD_RENDERER_OPENGL2=
++  BUILD_RENDERER_OPENGL2=0
+ endif
+ ifndef BUILD_AUTOUPDATER  # DON'T build unless you mean to!
+   BUILD_AUTOUPDATER=0
+@@ -62,7 +62,7 @@ ifeq ($(COMPILE_PLATFORM),cygwin)
+ endif
+ 
+ ifndef PLATFORM
+-PLATFORM=$(COMPILE_PLATFORM)
++PLATFORM=serenity
```

Comment:
Is this necessary with the change above?

Reply:
How do you mean?